fvm/evm/emulator/config.go

```diff
@@ -4,11 +4,13 @@ import (
 	"math"
 	"math/big"
 
-	"github.com/ethereum/go-ethereum/common"
-	"github.com/ethereum/go-ethereum/core"
-	"github.com/ethereum/go-ethereum/core/vm"
-	"github.com/ethereum/go-ethereum/crypto"
-	"github.com/ethereum/go-ethereum/params"
+	gethCommon "github.com/ethereum/go-ethereum/common"
```

Comment:
Good idea to add aliases to clarify these come from geth and aren't confused with Flow ones